vscode/webviews/components/EnhancedContextSettings.tsx

```diff
@@ -139,14 +140,23 @@ const CompactProviderComponent: React.FunctionComponent<{
     name: string
     inclusion: 'auto' | 'manual'
     handleRemove: (id: string) => void
-}> = ({ id, name, inclusion, handleRemove }): React.ReactNode => {
+    isIgnored: boolean
+}> = ({ id, name, inclusion, handleRemove, isIgnored }): React.ReactNode => {
     return (
         <div className={styles.enterpriseRepoListItem}>
-            <i className="codicon codicon-repo-forked" title={name} />
-            <span className={styles.repoName} title={name}>
+            {isIgnored ? (
+                <i className="codicon codicon-circle-slash" title={name} />
+            ) : (
+                <i className="codicon codicon-repo-forked" title={name} />
+            )}
```

Comment:
Thoughts on using `clsx` here too to change only the relevant part of the element?

```suggestion
            <i className={clsx('codicon', isIgnored ? 'codicon-circle-slash' : 'codicon-repo-forked'} title={name} />
```